app.rambox.Rambox.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: app.rambox.Rambox
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+rename-desktop-file: rambox.desktop
+rename-icon: rambox
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  ## necessary for webcam
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.portal.Background
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --filesystem=xdg-config/autostart:create
```

Comment:
What is it needed for?

Reply:
It has a autostart option in the settings. So it needs this permission to be able to add himself to autostart.

Reply:
I think it's a big breach of trust, effectively throwing any sandboxing promises away. It would be better to have that option broken than this.